lib/src/base-styles/_nx-tile.scss

```diff
@@ -95,19 +95,28 @@ $nx-tile-background-color: $nx-white;
 // Tile headers
 
 .nx-tile-header {
+  align-items: center;
+  display: grid;
+  display: -ms-grid;
+  grid-template-columns: 1fr auto;
+  -ms-grid-columns: 1fr auto;
   min-height: $nx-tile-header-height;
   margin: 8px 0 15px 0;
   overflow: hidden;
 }
 
 .nx-tile-header__title {
-  display: inline-block;
+  @extend .nx-truncate-ellipsis;
+  grid-column-start: 1;
+  grid-column-end: 1;
+  grid-row-start: 1;
+  grid-row-end: 1;
+  -ms-grid-row: 1;
+  -ms-grid-column: 1;
 
   h1, h2 {
-    display: inline-block;
+    @extend .nx-truncate-ellipsis;
```

Comment:
Do you need this mixin both here and on the parent?

Reply:
> Do you need this mixin both here and on the parent?

Frustratingly Yes. If it's only on `.nx-tile-header__title` it truncates but doesn't show the ellipsis. If it's only on the `<h#>` it doesn't truncate. If it's on both it truncates and shows the ellipsis.